src/zenml/zen_stores/migrations/versions/7d1919bb1ef0_add_run_templates.py

```diff
@@ -0,0 +1,102 @@
+"""Add run templates [7d1919bb1ef0].
+
+Revision ID: 7d1919bb1ef0
+Revises: b59aa68fdb1f
+Create Date: 2024-07-22 11:20:00.544451
+
+"""
+
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "7d1919bb1ef0"
+down_revision = "b59aa68fdb1f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
```

Comment:
Would it make sense here to merge the two migration scripts?

Reply:
Is there any real benefit? I think these are two separate features and much clearer if separated